src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -101,6 +101,7 @@
     private static final String ENV_TARANTOOL_WORKDIR = "TARANTOOL_WORKDIR";
     private static final String ENV_TARANTOOL_RUNDIR = "TARANTOOL_RUNDIR";
     private static final String ENV_TARANTOOL_DATADIR = "TARANTOOL_DATADIR";
+    private static final String ENV_INSTANCES_FILE = "INSTANCES_FILE";
```

Comment:
All existing env properties begin with ENV_TARANTOOL_ prefix. 
Lets follow the style and change to ENV_TARANTOOL_INSTANCES_FILE